rest_framework/filters.py

```diff
@@ -118,7 +118,10 @@ def filter_queryset(self, request, queryset, view):
 
     def to_html(self, request, queryset, view):
         cls = self.get_filter_class(view, queryset)
-        filter_instance = cls(request.query_params, queryset=queryset)
+        if cls:
```

Comment:
As a general note/nitpick can we go ahead and also make this look like the `filter_query()` method on top, basically just renaming `cls` for `filter_class`.